src/app/requestService/requestEditRoles.js

```diff
@@ -101,9 +101,11 @@ const post = async (req, res) => {
             let name = model.serviceRoles.filter((x) => x.id === item);
             displayroles.push(name[0].name);
           });
-          model.validationMessages.roles = `you have selected ${displayroles.map(
+          model.validationMessages.roles = `you have selected [${displayroles.map(
```

Comment:
The validation messages are usually sentence case - please doublecheck the requirements.

Reply:
changes to upper case Y